Marsiple/Common/Network/MartianApiManager.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  DataFetcher.swift
+//  Marsiple
+//
+//  Created by Matej Korman on 23/07/2018.
+//  Copyright © 2018 Matej Korman. All rights reserved.
+//
+
+import Foundation
+
+class MartianApiManager {
+    static func getPosts(success: @escaping ([Post]) -> Void, failure: @escaping (LocalizedError) -> Void) {
+        guard let url = MartianApi.url?.appendingPathComponent(MartianApi.URLs.posts)
+            else { return DispatchQueue.main.async { failure(DataFetcherError.urlCreationFailure) } }
+        getData(url: url, success: success, failure: failure)
+    }
+    
+    static func getComments(forPostId id: Int, success: @escaping ([Comment]) -> Void, failure: @escaping (LocalizedError) -> Void) {
+        guard let url = MartianApi.url?
+                            .appendingPathComponent(MartianApi.URLs.posts)
+                            .appendingPathComponent("\(id)")
+                            .appendingPathComponent(MartianApi.URLs.comments)
+            else { return DispatchQueue.main.async { failure(DataFetcherError.urlCreationFailure) } }
+        getData(url: url, success: success, failure: failure)
+    }
+    
+    static func postComment(comment: Comment) {
```

Comment:
Posting needs `success` and `failure` closures. We already talked about this one in another PR, is this resolved somewhere?

Reply:
We talked about it in this PR, but it was unclear what exactly I should do here, so I haven't resolved it yet, but I'm on it for next commit.

Reply:
Understood, can do.